example/android/app/google-services.json

```diff
@@ -0,0 +1,46 @@
+{
+  "project_info": {
+    "project_number": "864112465884",
+    "firebase_url": "https://apptentive-example-app.firebaseio.com",
+    "project_id": "apptentive-example-app",
+    "storage_bucket": "apptentive-example-app.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:864112465884:android:f009d8e2b2851a1c",
+        "android_client_info": {
+          "package_name": "com.apptentive.apptentive_flutter_example"
+        }
+      },
```

Comment:
one of these google-services.json files is a duplicate that's in the wrong location. Probably doesn't matter though.

Reply:
Thanks for catching this. This project had a number of merge conflicts when I pulled master back into develop, so that's probably where this crept in.

Reply:
Actually maybe Harry just added it twice. I think the one in the `app` directory is the right one.